.gitignore

```diff
@@ -134,8 +134,11 @@ dmypy.json
 /data/wikipedia-3sentence-level-retrieval-index/files/
 /pretrained/GPT-NeoX-20B/EleutherAI_gpt-neox-20b/
 /pretrained/Pythia-6.9B-deduped/EleutherAI_pythia-6.9b-deduped/
+/pretrained/RedPajama-3B/
 
 # ignore training output
 /model_ckpts/
 /huggingface_models/
 /training/wandb/
+
+*.jsonl
```

Comment:
Let's not blanket ignore `.jsonl` files. There likely will be other `.jsonl` files in this repo that we'll want to keep. Please follow the pattern above when ignoring data downloads.